pom.xml

```diff
@@ -27,9 +27,9 @@
         <version>6-SNAPSHOT</version>
     </parent>
 
-    <groupId>com.syncleus.maven.plugins</groupId>
-    <artifactId>mongodb-maven-plugin</artifactId>
-    <version>1.2.0-SNAPSHOT</version>
+    <groupId>com.github.agoston</groupId>
```

Comment:
These changes should not have been applied. If you fix it back to the original I can commit it (even if you dont ill fix it on my end when i get a chance in a day or so).

Reply:
Correct, I did not expect you to pick this up so swiftly, so I started using my fork, that is what you see here.
I'll roll back these changes now.